### ISSUE TITLE
Accessibility: html element should have a lang attribute

### ISSUE THREAD

**Issue by olbrich** [bullet-train-co/bullet_train-core#111]
https://github.com/bullet-train-co/bullet_train-core/blob/f87143fbc2dd41a5e97b76be22971f95fb22cbca/bullet_train-themes-light/app/views/themes/light/layouts/_account.html.erb#L2

According to accessibility standards, the html element should have a lang attribute with the appropriate language code.

Probably we can just add `lang="<%= I18n.locale %>"` to the `html` element.
